zkevm-circuits/src/evm_circuit/execution/precompiles/ec_add.rs

```diff
@@ -57,6 +59,12 @@ impl<F: Field> ExecutionGadget<F> for EcAddGadget<F> {
             cb.query_cell_phase2(),
             cb.query_cell_phase2(),
         );
+        let gas_cost = cb.query_cell();
```

Comment:
There is no need for a cell here. The constant expression could be used instead.